services/placement/options.go

```diff
@@ -20,16 +20,8 @@
 
 package placement
 
-import (
-	"github.com/m3db/m3cluster/services"
-	"github.com/m3db/m3x/instrument"
-)
-
 const (
 	defaultMaxStepSize = 3
-	defaultIsSharded   = true
-	// by default partial replace should be allowed for better distribution
-	defaultAllowPartialReplace = true
 )
 
 type deploymentOptions struct {
```

Comment:
Should the `deploymentOptions` live somewhere else e.g. `services/deploy/`? Does it have to live under `services/placement`?

Reply:
DeploymentOptions interface is defined in services/placement so I'd rather leave it here.